fast_carpenter/summary/binned_dataframe.py

```diff
@@ -116,6 +129,7 @@ def __init__(self, name, out_dir, binning, weights=None, dataset_col=False):
         variables, or a dictionary where the values are variables in the data and
         keys are the column names that these weights should be called in the
         output tables.
+      file_format (str or list[str], dict[str, str]): X
```

Comment:
I guess this was just a placeholder, but we should add a small description for what this does instead of `X`, how about: 
```
determines the file format to use to save the binned dataframe to disk.  Should be either a) a string with the file format, b) a dict containing the keyword `type` to give the file format and then all other keyword-argument pairs are passed on to the corresponding pandas function, or c) a list of values matching a) or b).
```

Reply:
Yeah i forgot to add in a description but that's great, i can add that.